<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>book_information</class>
 <widget class="QMainWindow" name="book_information">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1175</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>90</y>
      <width>247</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="10" column="1">
      <widget class="QTextEdit" name="intro_edit"/>
     </item>
     <item row="19" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="exit_but">
         <property name="text">
          <string>退出</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>出版社</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>作者</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="author_edit"/>
     </item>
     <item row="17" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="modify_but">
         <property name="text">
          <string>修改</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="insert_but">
         <property name="text">
          <string>添加</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delete_but">
         <property name="text">
          <string>删除</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item row="0" column="0" colspan="2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="search_edit"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="search_but">
         <property name="text">
          <string>搜索</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="name_but">
           <property name="text">
            <string>书名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="author_but">
           <property name="text">
            <string>作者</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="press_but">
           <property name="text">
            <string>出版社</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>简介</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>书名</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="press_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>书号</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="search_id">
         <property name="text">
          <string>查询</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>书号</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="id_edit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="search_id_edit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="ISBN_edit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>ISBN</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>藏书量</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="total_edit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>剩余量</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="left_edit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>等级</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="level_edit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="book_table">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>801</width>
      <height>491</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <column>
     <property name="text">
      <string>书号</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>ISBN</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>等级</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>书名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>作者</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>出版社</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>藏书量</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>剩余量</string>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
